### ISSUE TITLE
Accessible color contrast for nav items

### ISSUE THREAD

**Issue by cj213x** [liferay-design/liferay.design#763]
**Is your feature request related to a problem? Please describe.**
The selected state of the menu does not meet the standards for WCAG

**Describe the solution you'd like**
what if we put a selected state bar in the brand color and make the text white?

**Describe alternatives you've considered**
change the bg color of the nav bar

**Additional context**
![image](https://user-images.githubusercontent.com/26426085/89562015-332c5680-d7ce-11ea-86f3-c7ec5f7c2400.png)

**Comment by victorvalle** [comment]
Hi @cj213x !
Could you write down the steps to reproduce this bug?

Thanks :)

**Comment by plhnk** [comment]
@cj213x can you be more specific / include screenshots of which menu you're referencing?

**Comment by cj213x** [comment]
1. Navigate to handbook
2. Select a nav item

Result: Selected navigation item is not accessible.

![image](https://user-images.githubusercontent.com/26426085/90563917-599cab00-e159-11ea-9736-d185c47f7b3b.png)

**Comment by plhnk** [comment]
oooh thanks @cj213x I was thinking it was the main nav — what do you think about these ideas:

1. primary bg:
![image](https://user-images.githubusercontent.com/10226086/90566791-f19c9380-e15d-11ea-9843-82cf7d801f59.png)

2. darkened primary bg:
![image](https://user-images.githubusercontent.com/10226086/90566862-1133bc00-e15e-11ea-917f-9b10ab701326.png)

3. use hover bg, add primary line:
![image](https://user-images.githubusercontent.com/10226086/90567354-e138e880-e15e-11ea-9208-9117bd44f33f.png)

open to any others, mock preferred!

**Comment by plhnk** [comment]
@cj213x went w/#3 — take a look and let us know if this can be closed or if you'd like to try something else
